train.py

```diff
@@ -66,6 +83,7 @@ def train(dataset_name: str, num_workers: int = 0, is_test: bool = False, resume
     arg_parser.add_argument("--n_workers", type=int, default=0)
     arg_parser.add_argument("--test", action="store_true")
     arg_parser.add_argument("--resume", type=str, default=None)
+    arg_parser.add_argument("--model", type=str, default="code2seq")
```

Comment:
Let's set it as the first positional argument. Also, use choice to specify possible values (https://docs.python.org/3/library/argparse.html#choices)